projects/storefrontlib/shared/components/ng-select-a11y/ng-select-a11y.directive.ts

```diff
@@ -67,7 +73,19 @@ export class NgSelectA11yDirective implements AfterViewInit {
       this.featureConfigService.isEnabled('a11yNgSelectMobileReadout') &&
       inputElement.readOnly
     ) {
-      this.renderer.setAttribute(inputElement, 'aria-hidden', 'true');
+      const selectObserver = new MutationObserver((changes, observer) => {
```

Comment:
Couple points with this observer:
- We only start observing in mobile view. Should we create this observer in the breakpoint subscription too?
- If the user resizes their screen from desktop to below the breakpoint, would this observer be created? Should it be removed when returning to desktop size?

Reply:
The bug exists only for mobile devices' screen readers. It is device dependent, the behaviour should not changed if the screen is resized. This observer implementation should work, but maybe there is a better way to check for that? 🤔